projects/fuzzing-puzzles/project.yaml

```diff
@@ -0,0 +1,13 @@
+homepage: https://github.com/google/fuzzer-test-suite
+primary_contact: kcc@google.com
+auto_ccs:
+  - "kevinwkt@google.com"
+  - "kodewilliams@google.com"
+  - "metzman@google.com"
+  - "mmoroz@google.com"
+
+sanitizers:
+  - address
```

Comment:
maybe `undefined` instead?
the target aborts, so there is no reason to pay for asan instrumentation. 
(not too important though)

Reply:
Good point. Let me first check if my crappy code auto-generated from a decompiled binary doesn't make UBSan sad.